catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -551,7 +551,13 @@ public CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 .filter(Objects::nonNull)
                 .map(QueryResponse::getProperties)
                 .findFirst()
-                .orElse(Collections.emptyMap()));
+                .orElse(Collections.emptyMap()),
+            responses
+                .stream()
+                .filter(Objects::nonNull)
+                .map(QueryResponse::getProcessingDetails)
+                .findFirst()
```

Comment:
Not sure the context, but just making sure we are fine finding only the first and not using the rest? We don't need to collapse/reduce them into a single set?

Reply:
good catch - fixed